cmake/Platform/Linux/Packaging/prerm.in

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set -o errexit # exit on the first failure encountered
+
+{
+    if [[ -L "/usr/lib/x86_64-linux-gnu/libffi.so.6" ]]; then
+        sudo rm /usr/lib/x86_64-linux-gnu/libffi.so.6
+    fi
```

Comment:
Now this is even more dangerous.
What if the user actually installed libffi 6 on their own and are running their own version of python 3.7.X

Reply:
`libffi.so.6` wouldnt be a symlink

Reply:
You sure about that? because it is a symlink if is installed from a repo using apt-get
https://packages.ubuntu.com/bionic/amd64/libffi6/filelist

The actual file would be `/usr/lib/x86_64-linux-gnu/libffi.so.6.0.4`

Reply:
Honestly I think we should be creating the symlink in the first place for the user.
But if we are going to go down this path, we should remove it, as we don't know if the user has actually installed it on their own.

Reply:
you are right, will create a flag file instead

Reply:
I am OK with checking if the symlink is point to version 7.
Except ... version 8 is [out](https://packages.ubuntu.com/search?keywords=libffi8)

Reply:
I can always just not do anything... :) leaving the file around 
I can also check that the symlink doesn't point to a file that starts with the same, meaning, if it doesnt point to `libffi.so.6.X`, then I delete it

Reply:
Do nothing is probably the safest..